test/advection_test/CMakeLists.txt

```diff
@@ -1,233 +1,67 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.10)
-PROJECT(test_host)
-ENABLE_LANGUAGE(Fortran)
-
-include(CMakeForceCompiler)
-
-SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake/modules)
 
-#------------------------------------------------------------------------------
-#
-# Set where the CCPP Framework lives
-#
-#------------------------------------------------------------------------------
-get_filename_component(TEST_ROOT "${CMAKE_SOURCE_DIR}" DIRECTORY)
-get_filename_component(CCPP_ROOT "${TEST_ROOT}" DIRECTORY)
 #------------------------------------------------------------------------------
 #
 # Create list of SCHEME_FILES, HOST_FILES, and SUITE_FILES
 # Paths should be relative to CMAKE_SOURCE_DIR (this file's directory)
 #
 #------------------------------------------------------------------------------
-LIST(APPEND SCHEME_FILES "cld_suite_files.txt")
-LIST(APPEND SCHEME_FILES_ERROR "cld_suite_files_error.txt")
-LIST(APPEND HOST_FILES  "test_host_data" "test_host_mod")
-LIST(APPEND SUITE_FILES "cld_suite.xml")
-LIST(APPEND SUITE_FILES_ERROR "cld_suite_error.xml")
+set(SCHEME_FILES "cld_liq" "cld_ice" "apply_constituent_tendencies" "const_indices")
+set(SCHEME_FILES_ERROR "cld_liq" "cld_ice" "dlc_liq")
+set(HOST_FILES  "test_host_data" "test_host_mod")
+set(SUITE_FILES "cld_suite.xml")
+set(SUITE_FILES_ERROR "cld_suite_error.xml")
 # HOST is the name of the executable we will build.
 # We assume there are files ${HOST}.meta and ${HOST}.F90 in CMAKE_SOURCE_DIR
-SET(HOST "${CMAKE_PROJECT_NAME}")
-
-#------------------------------------------------------------------------------
-#
-# End of project-specific input
-#
-#------------------------------------------------------------------------------
+set(HOST "test_host")
 
-# By default, no verbose output
-SET(VERBOSITY 0 CACHE STRING "Verbosity level of output (default: 0)")
 # By default, generated caps go in ccpp subdir
-SET(CCPP_CAP_FILES "${CMAKE_BINARY_DIR}/ccpp" CACHE
-  STRING "Location of CCPP-generated cap files")
+set(CCPP_CAP_FILES "${CMAKE_CURRENT_BINARY_DIR}/ccpp")
 
-SET(CCPP_FRAMEWORK ${CCPP_ROOT}/scripts)
-
-# Use rpaths on MacOSX
-set(CMAKE_MACOSX_RPATH 1)
-
-#------------------------------------------------------------------------------
-# Set a default build type if none was specified
-if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-    #message(STATUS "Setting build type to 'Debug' as none was specified.")
-    #set(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build." FORCE)
-    message(STATUS "Setting build type to 'Release' as none was specified.")
-    set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
+# Create lists for Fortran and meta data files from file names
+list(TRANSFORM SCHEME_FILES       APPEND ".F90"  OUTPUT_VARIABLE SCHEME_FORTRAN_FILES)
```

Comment:
Where are TRANSFORM_SCHEME_FILES et al defined?

Reply:
What a difference one underscore makes ... sorry for not reading this correctly.